macos/RNCPicker.m

```diff
@@ -0,0 +1,174 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#import "RNCPicker.h"
+
+#import <React/RCTConvert.h>
+#import <React/RCTUtils.h>
+
+
+@interface RNCPicker() <NSComboBoxDataSource, NSComboBoxDelegate>
+@end
+
+@implementation RNCPicker
+
+- (instancetype)initWithFrame:(CGRect)frame
+{
+    if ((self = [super initWithFrame:frame pullsDown:false])) {
+        _color = [NSColor blackColor];
+        _font  = [NSFont systemFontOfSize:21];
+        _selectedIndex = NSNotFound;
+        _textAlign     = NSTextAlignmentCenter;
+        //self.dataSource = self;
+        //self.delegate =  self;
+        //self.completes = true;
+        [self selectItemAtIndex:0];
+        [[self menu] setFont:_font];
+        self.pullsDown = false;
+        [self setAction:@selector(mySelector:)];
+        [self setTarget:self];
+
+    }
+    return self;
+  /*if ((self = [super initWithFrame:frame])) {
+    _color = [UIColor blackColor];
+    _font = [UIFont systemFontOfSize:21]; // TODO: selected title default should be 23.5
+    _selectedIndex = NSNotFound;
+    _textAlign = NSTextAlignmentCenter;
+    self.delegate = self;
+    self.dataSource = self;
+    [self selectRow:0 inComponent:0 animated:YES]; // Workaround for missing selection indicator lines (see https://stackoverflow.com/questions/39564660/uipickerview-selection-indicator-not-visible-in-ios10)
+  }
+  return self;*/
```

Comment:
if there a reason for this comment to be here?
if so, could you add a comment in the code :+1: